pkg/cloudprovider/suite_test.go

```diff
@@ -1135,7 +1135,7 @@ var _ = Describe("CloudProvider", func() {
 			ExpectProvisioned(ctx, env.Client, cluster, cloudProvider, prov, pod)
 			ExpectScheduled(ctx, env.Client, pod)
 			createFleetInput := awsEnv.EC2API.CreateFleetBehavior.CalledWithInput.Pop()
-			Expect(fake.SubnetsFromFleetRequest(createFleetInput)).To(ConsistOf("test-subnet-2"))
+			Expect(fake.SubnetsFromFleet(createFleetInput)).To(ConsistOf("test-subnet-2"))
```

Comment:
Why change the name of these functions?

Reply:
Per the migration documents, The <OperationName>Request, <OperationName>WithContext, and <OperationName> methods have all been consolidated into single operation method, <OperationName>.